tox.ini

```diff
@@ -1,7 +1,11 @@
 [tox]
-envlist = py27,py33
+envlist = django{18,19,110}
```

Comment:
python version list not needed in tox.ini ? the definition in travis.yml is enough ?